cli/pkg/kctrl/cmd/package/available/generate-default-values.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020 VMware, Inc.
```

Comment:
file names do not have dashes.

Reply:
Hmm, I thought I was using underscore. Good catch 👍🏻